### PR TITLE
Fix/coverage

### LINKED ISSUE #2039: MAM: handle a case when "keys" folder wasn't cleaned
**Is your feature request related to a problem? Please describe.**
Currently if MAM wasn't correctly closed (by calling `stop_manager` method) it will fail on start with `FileExists` error because `keys` folder wasn't erased. That means that's impossible to restart MAM after force exit, only by manually deleting a `keys` folder.

**Describe the solution you'd like**
Make MAM able to start with `keys` folder present (by default or with some flag) and handle that case.

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes

fix for manager. coverage fixes.

## Fixes

#2039 

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have checked that the documentation about the `aea cli` tool works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/test_cli/test_generate/test_generate.py [hunk 1]
```diff
@@ -90,7 +87,7 @@ def test__generate_item_no_res_isort_missing(self, *_mocks):
         )
         self.assertEqual(cm.exception.message, expected_msg)
 
-    @pytest.mark.skip
+    # @pytest.mark.skip
```
Comment (DavidMinarsch) [inline tests/test_cli/test_generate/test_generate.py:90]: remove

File: aea/cli/fetch.py [hunk 2]
```diff
@@ -137,6 +137,10 @@ def fetch_agent_locally(
         )
 
     # add dependencies
+    if not ctx.config.get("is_local") or not ctx.config.get("is_mixed"):
+        # used by add_item to download locally
+        ctx.config["is_local"] = True
```
Comment (DavidMinarsch) [inline aea/cli/fetch.py:142]: By definition we are in the local case here; so if anything this should be set at the start of the method. But preferably; this is correctly set outside this method and then only asserted at the start.

Comment (marcofavorito) [reply aea/cli/fetch.py:142]: yes. If not necessary, I'd put this outside and put the asserts at the start of the function

**Review by panasevychol — APPROVED** [review]
My approval to MAM fix

File: tests/test_cli/test_generate/test_generate.py (hunk 1 above)
Comment (solarw) [reply tests/test_cli/test_generate/test_generate.py]: removed

File: aea/cli/fetch.py (hunk 2 above)
Comment (solarw) [reply aea/cli/fetch.py]: updated, check is added

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**